plugins/regex-actions/src/actions/regex/replace.test.ts

```diff
@@ -20,7 +20,7 @@ describe('regex:replace', () => {
 
   const logStream = {
     write: jest.fn(),
-  } as jest.Mocked<Partial<Writable>> as jest.Mocked<Writable>;
+  } as any as Writable;
```

Comment:
The 

```tsx
    await action.handler(context);
```

in line 63 needs an `ActionContext` from:

```
import { ActionContext } from '@backstage/plugin-scaffolder-node';
```

```tsx
/**
 * ActionContext is passed into scaffolder actions.
 * @public
 */
export declare type ActionContext<TActionInput extends JsonObject, TActionOutput extends JsonObject = JsonObject> = {
    logger: Logger;
    logStream: Writable;
...
```

And this `logStream: Writeable` doesn't accept a `jest.Mock`.

I can also change the `mockContext` to simple:

```tsx
    logStream: jest.fn() as any as Writable,
```

Or:

```tsx
    logStream: {} as Writable,
```

Both works fine. It looks like the given logStream isn't used... Let me pick up the latest option so that it's a more straightforward cast, and it would then crash in the future if one of the `logStream` properties were used internally.